selfdrive/car/mazda/interface.py

```diff
@@ -40,8 +40,26 @@ def _get_params(ret, candidate, fingerprint, car_fw, experimental_long):
       ret.mass = 3443 * CV.LB_TO_KG + STD_CARGO_KG
       ret.wheelbase = 2.83
       ret.steerRatio = 15.5
+    elif candidate in CAR.MAZDA3_2019:
+      ret.openpilotLongitudinalControl = True
+      ret.stopAccel = 0.0
+      ret.longitudinalTuning.kpBP = [0., 5., 35.]
+      ret.longitudinalTuning.kpV = [1.2, 0.8, 0.5]
+      ret.longitudinalTuning.kiBP = [0., 35.]
+      ret.longitudinalTuning.kiV = [0.18, 0.12]
```

Comment:
Let's do longitudinal in a subsequent PR so this one is more easily mergable

Reply:
Ok sounds good.